package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azurestaticwebapps",
     "displayName": "Azure Static Web Apps",
     "description": "%staticWebApps.description%",
-    "version": "0.11.1-alpha.1",
+    "version": "0.11.1",
```

Comment:
The main reason we don't want to change this version yet is because 

a.) We might try to include more fixes before we release
b.) When CTI tests our builds, it'll create telemetry entries based off of that. We don't want to start collecting telemetry because this might not be what actually gets released in the wild